More flexibel Serverless deploy configuration

martijngastkemper opened the pull request:
- Describe how to deploy with function URLs
- Describe how to deploy with your own Serverless configuration
- Remove ListBucket action, the function doesn't need it

martijngastkemper commented:
After reviewing it I will merge this change in both v3.x and v4.x.

harmenjanssen reviewed (approved):
LGTM! Ik ga hier zo doorheen voor de GRRR site.

martijngastkemper committed:
Add documentation about using your own Serverless config

- Discribe how to deploy with function URLs
- Remove ListBucket action, the function doesn't need it
